.github/workflows/main.yml

```diff
@@ -0,0 +1,87 @@
+name: main
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        name: [
+          "py-27",
+          "py-33",
+          "py-35",
+        ]
```

Comment:
What about the other Python versions?

Reply:
Is still in progress, I will add the other Python versions. 
I converted the PR to draft.